src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsactivitiesmanagementapi/service/ManageAttendancesService.kt

```
@@ -83,6 +83,8 @@ class ManageAttendancesService(
                 log.info("Sending sync event for attendance ID ${saved.attendanceId} ${saved.prisonerNumber} ${saved.scheduledInstance.activitySchedule.description}")
                 outboundEventsService.send(OutboundEvent.PRISONER_ATTENDANCE_CREATED, saved.attendanceId)
               }
+            }.onFailure {
+              log.error("Error occurred saving attendances", it)
```

I noticed we are potentially swallowing errors here without even knowing about it.

Given we effectively have a try catch here I did wonder if we should also be raising it via the monitoring service or do away with the try catch and let it propagate up the stack?

---

Would it continue after the failure? i.e. just affect one attendance row, or one activity? If they're all failing, not sure we want to be spammed with 1300 alerts?  The 5-mile-island problem!